code/datums/abnormality/datum/abnormality.dm

```diff
@@ -163,7 +163,17 @@
 		return 0
 	var/acquired_chance = available_work[workType]
 	if(islist(acquired_chance))
-		acquired_chance = acquired_chance[get_user_level(user)]
```

Comment:
For example, if you don't want to make it a proc of its own:
`var/work_level = clamp(round(get_attribute_level(user, work_type)/20), 1, 5)`
`acquired_chance = acquired_chance[work_level]`

Reply:
Wait, I'm dumb. There does, however, exist a define to turn work type names into attribute names. Use it.